oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOAuth2ResourceServerAutoConfig.java

```diff
@@ -46,10 +46,7 @@
 
 import java.net.InetSocketAddress;
 import java.net.Proxy;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Is wildcard import recommended?

Reply:
IntelliJ auto-refactored the imports and put this wildcard in :)

Reply:
Fixed